### DISCUSSION TITLE
exec /usr/local/bin/docker-entrypoint.sh: exec format error

### DISCUSSION THREAD

**Discussion by jokay** [renovatebot/renovate#26601] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.128.2

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

It works on the `amd64` architecture but when running on the `arm64` architecture it fails.

Minimal repro is difficult as it's self-hosted and not exposed to the internet at all.

### Relevant debug logs

GitLab runner (**amd64**) works

<details><summary>Logs</summary>
Using docker image sha256:7617a67efefad249eb5f89ef114793aa94c997c3dda9662d54c7cbf93cb4a8c4 for renovate/renovate:slim with digest renovate/renovate@sha256:332317121135188bba6a01803984893ea3876de8b82a4681936fc4d8c476c73e ...
...
Job succeeded
</details>

GitLab runner (**arm64**) does not work

<details><summary>Logs</summary>
Using docker image sha256:7617a67efefad249eb5f89ef114793aa94c997c3dda9662d54c7cbf93cb4a8c4 for renovate/renovate:slim with digest renovate/renovate@sha256:332317121135188bba6a01803984893ea3876de8b82a4681936fc4d8c476c73e ...
exec /usr/local/bin/docker-entrypoint.sh: exec format error
Cleaning up project directory and file based variables 00:00
ERROR: Job failed: exit code 1
</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by jokay** [comment]
Or is a container for **arm64** *(not yet / no longer)* available? @viceice 

![image](https://github.com/renovatebot/renovate/assets/18613935/cd070e88-36d7-4cb0-bf2f-782452644410)

**Reply by viceice** [reply, marked as answer]
don't use slim tags, slim tags are deprecated and amd64 only. use latest tag or a version.
